config/services.yaml

```diff
@@ -46,13 +48,24 @@ services:
 
     ### Event Store
 
+
+    App\Infrastructure\Event\IterableDbalEventStore:
```

Comment:
Why duplicate the event store?

If we need a feature that it's not implemented in broadway I prefer one of this options:

- Extend and implement the method we need.
- Create our own event store and encapsulate the broadway event store and implement the features we need.

Duplicate the event store for a single feature seems not the way to go . ;)